MekHQ/src/mekhq/campaign/mission/AtBScenario.java

```diff
@@ -1635,6 +1638,19 @@ protected void writeToXmlEnd(PrintWriter pw1, int indent) {
 
             pw1.println(MekHqXmlUtil.indentStr(indent+1) + "</transportLinkages>");
         }
+        
+        if (numPlayerMinefields.size() > 0) {
+            pw1.println(MekHqXmlUtil.indentStr(indent+1) + "<numPlayerMinefields>");
+
+            for (int key : numPlayerMinefields.keySet()) {
+                pw1.println(MekHqXmlUtil.indentStr(indent+2) + "<numPlayerMinefield>");
+                MekHqXmlUtil.writeSimpleXmlTag(pw1, indent + 3, "minefieldType", key);
+                MekHqXmlUtil.writeSimpleXmlTag(pw1, indent + 3, "minefieldCount", numPlayerMinefields.get(key).toString());
+                pw1.println(MekHqXmlUtil.indentStr(indent+2) + "</numPlayerMinefield>");
+            }
+
+            pw1.println(MekHqXmlUtil.indentStr(indent+1) + "</numPlayerMinefields>");
+        }
```

Comment:
I thought we had a routine to SerDe simple dictionaries?

Reply:
Yeah, I think we do, I just can't for the life of me remember where the hell it is.